docs/plugins.md

```diff
@@ -108,25 +96,20 @@ Go-ipfs supports two types of plugins: External and Preloaded.
 
 ### External Plugin
 
-The advantage of an external plugin is that it can be built, packaged, and
-installed independently of go-ipfs. Unfortunately, this method is only supported
-on Linux and MacOS at the moment. Users of other operating systems should follow
-the instructions for preloaded plugins.
+The advantage of an external plugin is that it can be built, packaged, and installed independently of go-ipfs. Unfortunately, this method is only supported on Linux and MacOS at the moment. Users of other operating systems should follow the instructions for preloaded plugins.
 
 #### In-tree
 
-To build plugins included in
-[plugin/plugins](https://github.com/ipfs/go-ipfs/tree/master/plugin/plugins),
-run:
+To build plugins included in [plugin/plugins](https://github.com/ipfs/go-ipfs/tree/master/plugin/plugins), run:
 
-```bash
+```console
```

Comment:
Let's just switch to bash everywhere. It gives us some nice syntax highlighting.

Reply:
Okay, I just used console because that was on the first file I changed :)